annotations/fenix/README.md

```diff
@@ -0,0 +1,6 @@
+---
+featured: true
```

Comment:
what is a "featured" app?

Reply:
Displayed first on the front page 😄

Reply:
I'll add some comments to make this more clear.